packages/tempo/bin/push-docs.sh

```
@@ -0,0 +1,49 @@
+echo "Pulling latest main..."
+git pull --ff-only origin main
+
+echo "Applying doc changes from $CURRENT_BRANCH to main..."
+# Checkout only the doc files from the branch
+git checkout $CURRENT_BRANCH -- $DOC_PATHS
+
+# Check if there's actually anything to commit
+if git diff-index --quiet HEAD --; then
+  echo "No doc changes found between main and $CURRENT_BRANCH."
+  echo "Switching back to $CURRENT_BRANCH..."
+  git checkout $CURRENT_BRANCH
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)